src/main/java/com/ericsson/eiffelcommons/subscriptionobject/SubscriptionObject.java

```diff
@@ -30,6 +30,16 @@ public SubscriptionObject(final Class<T> selfClass, String subscriptionName) thr
         this.self = selfClass.cast(this);
     }
 
+    /**
+     * Adds a notification body to the subscriptionObject.
+     *
+     * @param notificationBody
+     * @return
```

Comment:
what is the return here, can that be added to the javadoc.

Reply:
It returns itself, I added the class type